app/views/users/backup_code_setup/index.html.erb

```diff
@@ -18,5 +18,3 @@
     <%= link_to @presenter.other_option_title, two_factor_options_path %>
   <% end %>
 </div>
-
-<%= javascript_packs_tag_once('webauthn-unhide-signup') %>
```

Comment:
including this script goes back a bit, but it's not clear why it would be needed on the backup code page, so I've removed it

original PR: https://github.com/18F/identity-idp/pull/3044